### ISSUE TITLE
Add a way to filter by queries and query classes

### ISSUE THREAD

**Issue by cH3n7i** [hyrise/Cockpit#441]
**Please describe the solution you'd like:**  
Please give some more insight about this idea @Bensk1 

**Affected Component(s):**  
`Frontend`

**Comment by Bensk1** [comment]
We do not have an exact plan for that, yet.

For detailed analysis it could be useful to focus on a few queries, e.g., TPC-H queries 6, 9, 12. However, looking at the join order benchmark, offering 100 possible filter options is painful and ugly. A possible solution could be to select certain query classes. Such a (latency-based) classification was part of one of your preparation assignments. 

Any thoughts on that @Bouncner?

**Comment by Bouncner** [comment]
I'd like both (expandable lists of filters, the 100 query should not annoy users that do not want to filter on single queries). So we can filter by EITHER (i) query groups (defined statically in a JSON) and (ii) single queries.

Does that make sense?

**Comment by MaxSchneider1337** [comment]
@Bensk1 @Bouncner this should be fixed by #539 (?)

**Comment by Bouncner** [comment]
Nope, as far as I know we neither support query classes (a set a queries) nor can we get througput for a subset of queries. Right?
